packages/plugin-legacy/package.json

```diff
@@ -4,11 +4,24 @@
   "license": "MIT",
   "author": "Evan You",
   "files": [
-    "index.js",
-    "index.d.ts"
+    "dist"
   ],
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
```

Comment:
Although not necessary, it's good to move `types` above `require` (first in the object) as resolving `types` should technically take precedence over `import` and `require`.

IIRC TypeScript had implemented a non-standard `exports` resolver to make this unnecessary, but I think it's good that we follow the resolver spec.

Reply:
How the order affects them? I do this because it follows what we used to have "main", "module", "types" on the top-level

Reply:
Really? What? 🤯 It feels very unintuitive. But anyway I don't mind to do that

Reply:
As a reference point, [Svelte](https://github.com/sveltejs/svelte/blob/master/package.json#L38) also has it at the top.

Reply:
Done